CHANGELOG.md

```diff
@@ -6,6 +6,9 @@ description: |-
 ---
 
 # VKCS Provider's changelog
+
+#### v0.7.3 (unreleased)
```

Comment:
There is no need to add empty changelog, even if CI check will not pass.

Reply:
this makes sense at the moment when no unreleased changes are registered there - this dramatically reduces count of running tests

Reply:
remove v0.7.3